wordpress-mobile/WordPress-Android
# Make the IME Submit button work fullscreen reply dialog in landscape orientation

Fixes #11449 

This PR fixes the non-working send button in expanded comment reply/edit dialog in landscape mode.
[![Image from Gyazo](https://i.gyazo.com/6a5c54037e50f7276361fce5738883e8.png)](https://gyazo.com/6a5c54037e50f7276361fce5738883e8)

To test:
- Turn the device to landscape mode.
- Open a comment.
- Expand the comment reply.
- Click Comment field.
- The soft keyboard should be shown now; enter a comment.
- Press SEND confirm that reply is published.

PR submission checklist:
